pages/news.vue

```diff
@@ -1,32 +1,30 @@
 <template>
   <v-container>
+    <h1 class="mt-3"> 
```

Comment:
let's remove this top margin class? We should rely on the default margin spacing given and only extra margin spacing if necessary; `<h1>` styling already comes with top margin spacing.